package.json

```diff
@@ -25,10 +26,25 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "husky": "^4.3.0",
     "jest": "^24.9.0",
+    "lint-staged": "^10.4.0",
     "prettier": "^2.1.2"
   },
   "engines": {
     "node": "10.* || 12.* || >= 14"
+  },
+  "lint-staged": {
+    "*.{js,json}": [
+      "prettier --write"
```

Comment:
Let's remove, I generally dislike things in precommit modifying the repos files (having the `eslint` in there should provide the same value of preventing non-prettiered stuff from flowing through to the commit though).

Reply:
That's fair, I'll remove prettier but leave eslint on precommit.